src/calculateRentalCost.js

```diff
@@ -1,10 +1,23 @@
 /**
+ *
  * @param {number} days
  *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const cost = 40;
+
+  if (days < 3) {
+    return cost * days;
+  }
+
+  if (days < 7) {
```

Comment:
The condition `if (days < 7)` is correct, but since the previous condition already checks for `days < 3`, this condition can be simplified to `else if (days < 7)` for better readability and to avoid unnecessary checks.